hawkular-nest/hawkular-nest-feature-pack/pom.xml

```diff
@@ -96,6 +96,100 @@
   <build>
     <plugins>
 
+      <!-- Unpack the upstream artifacts that host standalone*.xml stubs that we want to adapt in the next steps -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>unpack-deps</id>
+            <phase>generate-resources</phase>
+            <goals>
+              <goal>unpack</goal>
+            </goals>
+            <configuration>
+              <artifactItems>
+                <artifactItem>
+                  <groupId>org.hawkular.commons</groupId>
+                  <artifactId>hawkular-bus-feature-pack</artifactId>
+                  <type>zip</type>
+                  <includes>configuration/standalone/template.xml,configuration/standalone/subsystems.xml</includes>
+                  <outputDirectory>${project.build.directory}/feature-pack-resources-tmp</outputDirectory>
+                </artifactItem>
+              </artifactItems>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+
+      <!-- Generate the standalone*.xml stubs from the upstream originals using XSL transformations -->
```

Comment:
> One transformation set is for subsystems.xml , the other is for template.xml.

Strictly speaking yes, but taken a little bit figuratively, both subsystems.xml  and template.xml are some kind of a stub used for building standalone*.xml files, are they not?